src/Cake.Common/Tools/DupFinder/DupFinderSettings.cs

```diff
@@ -91,5 +91,15 @@ public sealed class DupFinderSettings : ToolSettings
         /// Gets or sets a value indicating whether to show duplactes text in the report.
         /// </summary>
         public bool ShowText { get; set; }
+
+        /// <summary>
+        /// Gets or sets an optional flag for if should throw exception on finding duplicates
+        /// </summary>
+        public bool? ThrowOnDuplicates { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether analysis of results file should be completed
+        /// </summary>
+        public bool PerformResultsFileAnalysis { get; set; }
```

Comment:
@DevWouter yip, that makes sense.  Based on some of the other feedback, this property may actually not even be required, but I will decide that when I look again at this, which will hopefully be tonight.